tests/test_hf_api.py

```diff
@@ -44,10 +44,10 @@
 )
 from huggingface_hub.community import DiscussionComment, DiscussionWithDetails
 from huggingface_hub.constants import (
-    REPO_TYPE_DATASET,
-    REPO_TYPE_MODEL,
-    REPO_TYPE_SPACE,
-    SPACES_SDK_TYPES,
+    REPO_TYPE_DATASET,  # noqa: F401 # kept for backward compatibility
+    REPO_TYPE_MODEL,  # noqa: F401 # kept for backward compatibility
+    REPO_TYPE_SPACE,  # noqa: F401 # kept for backward compatibility
+    SPACES_SDK_TYPES,  # noqa: F401 # kept for backward compatibility
```

Comment:
no need to keep backward compatibility in tests since tests are never imported from another library